README.md

```diff
@@ -8,17 +8,13 @@ The snap is **NOT SUPPORTED** by the OpenThread Border Router community.
 
 OpenThread Border Router source code: https://github.com/openthread/ot-br-posix
 
-## Build and Install
-Execute the following command from the top-level directory of this repo:
+## Install
+The snap can be installed from the store:
 ```bash
 snapcraft -v
 ```
```

Comment:
Installation command corrected in https://github.com/canonical/openthread-border-router-snap/pull/22